test/snippet/search/dream_index/interleaved_bloom_filter_get.cpp

```diff
@@ -0,0 +1,17 @@
+#include <seqan3/search/dream_index/interleaved_bloom_filter.hpp>
+#include <seqan3/core/debug_stream.hpp>
```

Comment:
#include <seqan3/core/debug_stream.hpp> for #include <seqan3/search/dream_index/interleaved_bloom_filter.hpp> ?